docs/variants/protectli_vp2420/releases.md

```diff
@@ -12,7 +12,48 @@ For details about our release process please read
 </center>
 
 Test results for this platform can be found
-[here](https://docs.google.com/spreadsheets/d/1wI0qBSLdaluayYsm_lIa9iJ9LnPnCOZ9eNOyrKSc-j4/edit#gid=975611333).
+[here](https://docs.google.com/spreadsheets/d/1wI0qBSLdaluayYsm_lIa9iJ9LnPnCOZ9eNOyrKSc-j4/edit#gid=579117128).
+
+## v1.1.0 - 2023-04-20
+
+### Added
+
+- [USB stack and mass storage enable/disable option](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#usb-configuration)
+- [SMM BIOS write protection enable/disable option](https://docs.dasharo.com/dasharo-menu-docs/dasharo-system-features/#dasharo-security-options)
```

Comment:
@Pokisiekk it's recommended to use relative links to the documents inside the `mkdocs`. Main benefit is to get clear messages in the `mkdocs serve` logs, if any links are broken inside the service. Using direct links, there is no such feature. Could you fix links in this doc?

ex.:
```
- [USB stack and mass storage enable/disable option](../../dasharo-menu-docs/dasharo-system-features.md#usb-configuration)
```

Reply:
Yes it was generated by template, so I doubt we have any influence on that. Of course we can purposefully fill the templates with relative URLs for those we know can be relative...